httpcore/_types.py

```diff
@@ -9,4 +9,5 @@
 Origin = Tuple[bytes, bytes, int]
 URL = Tuple[bytes, bytes, Optional[int], bytes]
 Headers = List[Tuple[bytes, bytes]]
+SocketAddr = Tuple
```

Comment:
Anything more specific?

Reply:
It varies.. you can receive Tuple[str, int] or Tuple[str, int, int, int]